routes.js

```diff
@@ -76,15 +80,15 @@ module.exports = (app, logger) => {
         'more-projects': body['more-projects'],
         'events': body.events,
         'more-events': body['more-events']
-      }).save().then(survey => {
+      }).save().then((survey) => {
         user.description = survey
         return user.save()
       })
-    }).then(user => {
+    }).then((user) => {
       // Successful save and invitation
       logger.info('Successfully saved user')
-      next()
-    }, err => {
+      next() // Move to addToSlack
+    }, (err) => {
       console.log(err)
       if (err.code === 11000) {
```

Comment:
Checking for this duplicate key error seems unnecessary now with `findOneAndUpdate `